packages/workbox-routing/src/Router.ts

```diff
@@ -128,11 +130,10 @@ class Router {
 
         // If a MessageChannel was used, reply to the message on success.
         if (event.ports && event.ports[0]) {
-          await requestPromises;
-          event.ports[0].postMessage(true);
+          requestPromises.then(() => event.ports[0].postMessage(true));
```

Comment:
Declaring the listener function as being an `EventListener` wouldn't work if it was `async`, since `EventListener`s aren't supposed to return `Promise<void>`.

So I just took the one `await` and converted it into a `then()`, and dropped the `async` from the listener function declaration.